cmd/yawol-cloud-controller/main.go

```diff
@@ -121,7 +121,9 @@ func main() {
 	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
 
-	classNames = append(classNames, helper.DefaultLoadbalancerClass)
+	if len(classNames) == 0 {
+		classNames = append(classNames, helper.DefaultLoadbalancerClass)
```

Comment:
Nit: I think, you can simply initialize the variable with `[]string{helper.DefaultLoadbalancerClass}` before defining the flag.

Reply:
Nvm, this won't work with the `append`-based implementation of the `Value` interface